src/lib/containers/evaluation_queues/CalendarWithIconFormGroup.tsx

```diff
@@ -26,19 +33,22 @@ export const CalendarWithIconFormGroup: React.FunctionComponent<
         {label && <label htmlFor={id}>{label}</label>}
         <InputGroup>
           <DateTimePicker
+            className="datetime-picker"
             value={value}
             disabled={disabled}
             onChange={setterCallback}
             renderInput={params => (
               <TextField
                 disabled={disabled}
                 id={id}
-                className="form-control calendar-date-time-input rounded-right"
                 variant="outlined"
                 {...params}
               />
             )}
           />
+          <span style={{ margin: 'auto 8px' }}>
+            {dayjs().tz(dayjs.tz.guess()).format('z')}
+          </span>
```

Comment:
Is it possible to change the DateTimePicker and this rendered text if the user has UTC mode on?

Reply:
Actually...was the old picker always in local time? This is probably fine if so